reservations/templates/reservations/reservation_details.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<html lang-"en">
```

Comment:
`lang="en"` - I think you misread this from my screen share.

Reply:
Got it fixed on all html docs

Reply:
ty bro